libmachine/ssh/client.go

```diff
@@ -379,6 +389,12 @@ func NewExternalClient(sshBinaryPath, user, host string, port int, auth *Auth) (
 }
 
 func getSSHCmd(binaryPath string, args ...string) *exec.Cmd {
+	// remove the quote to avoid parsing errors
+	for i, arg := range args {
+		if strings.HasPrefix(arg, "ProxyCommand='") {
+			args[i] = strings.ReplaceAll(arg, "'", "")
```

Comment:
If I understand correctly, we have to include the quote in the proxy string above so that the `args` get split correctly. If I understand this correctly, then here can we `break` after we find the `ProxyCommand` to stop the loop?

Reply:
You are right. 
A break is added.